kindtest/workflows/job-failure.tmpl.yaml

```diff
@@ -7,6 +7,6 @@ jobs:
     runs-on: [self-hosted, "{{.Namespace}}/{{.RunnerPool}}"]
     steps:
       - run: job-started
-      - run: false
-      - if: failure()
-        run: job-failure
+      # /tmp/test is created by a setup command in runnerpool2.
+      - run: cat /tmp/test
```

Comment:
I am uncomfortable with checking if job-failure is being executed in a workflow that is expected to succeed.

Reply:
Thank you for your comment.
In my opinion, the purpose of the `job-success/failure` workflow is to check the runner pod's behavior when `job-success/failure` is called.
So we need not fail a job on purpose. 

By the way, it was not good to have two checks in the `job-failure` workflow.
So I make another workflow `setup-command`.

Reply:
Now the kindest implementation is redundant.
So I want to refactor it in another PR.